### PR TITLE
Refresh issue templates based on UCL-MIRSG/MIRSG#186

### PR DESCRIPTION
**Pull request by HChughtai**
Updated issue templates based on the issue types being described in UCL-MIRSG/MIRSG#186

Markdown templates have been replaced with [GitHub issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)

Comments and feedback on the content of each gratefully received

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/problem.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+name: Problem
+description: Track anything that is broken, wrong or needs fixing
+labels:
+  - problem
+projects:
+  - UCL-MIRSG/3
+body:
+  - type: markdown
+    attributes:
+      value: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/problem.yaml:11]:
```suggestion
      value: |-
```

File: .github/ISSUE_TEMPLATE/problem.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      label: Short Description
+      description: Briefly summarise the problem.
+      placeholder: What is the problem? What is the impact?
+    validations:
+      required: true
+  - type: textarea
+    id: evidence
+    attributes:
+      label: Evidence/Steps to Reproduce
+      description: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/problem.yaml:26]: > 
```suggestion
      description: |-
```

File: .github/ISSUE_TEMPLATE/problem.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        Can you reproduce the problem? If so, what are the steps to reproduce it?
+        Otherwise, provide any evidence that helps to understand the problem.
+      placeholder: e.g. List of steps, screenshots, logs, error messages, etc.
+    validations:
+      required: false
+  - type: textarea
+    id: acceptance_criteria
+    attributes:
+      label: Acceptance Criteria/Expected Behaviour
+      description: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/problem.yaml:36]:
```suggestion
      description: |-
```

File: .github/ISSUE_TEMPLATE/problem.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    attributes:
+      value: |
+        Thanks you for raising this problem!
+        ---
+        Fill in the below once the problem is resolved:
+  - type: textarea
+    id: resolution
+    attributes:
+      label: Resolution
+      description: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/problem.yaml:58]:
```suggestion
      description: |-
```

File: .github/ISSUE_TEMPLATE/problem.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        What was done to resolve the problem?
+        What was the root cause? Reference any related issues or pull requests.
+      placeholder: "e.g. 'The problem was resolved by doing x, y, z in #123'"
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/problem.yaml:61]: Consistent with elsewhere
```suggestion
      placeholder: e.g. "The problem was resolved by doing x, y, z in #123"
```

File: .github/ISSUE_TEMPLATE/question.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+name: Question
+description: Ask a question, seek advice or ask for feedback
+labels:
+  - question
+projects:
+  - UCL-MIRSG/3
+body:
+  - type: markdown
+    attributes:
+      value: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/question.yaml:11]:
```suggestion
      value: |-
```

File: .github/ISSUE_TEMPLATE/question.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        Thanks for asking this question!
+        ---
+        Fill in the below once the question is answered:
+  - type: textarea
+    id: next_steps
+    attributes:
+      label: Next Steps/Accepted Response
+      description: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/question.yaml:40]:
```suggestion
      description: |-
```

File: .github/ISSUE_TEMPLATE/question.yaml [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+        What was the answer to the question? What are the next steps?
+        Reference any related issues or pull requests.
+      placeholder: "e.g. 'We decided to do x, y, z in #167 to address this question'"
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/question.yaml:43]:
```suggestion
      placeholder: e.g. "We decided to do x, y, z in #167 to address this question"
```

File: .github/ISSUE_TEMPLATE/work.yaml [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+name: Work
+description: Track any work to be done
+labels:
+  - work
+projects:
+  - UCL-MIRSG/3
+body:
+  - type: markdown
+    attributes:
+      value: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/work.yaml:11]:
```suggestion
      value: |-
```

File: .github/ISSUE_TEMPLATE/work.yaml [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+      label: Short Description
+      description: Briefly summarise the work to be done
+      placeholder: What needs to be done?
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance_criteria
+    attributes:
+      label: Acceptance Criteria
+      description: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/work.yaml:26]:
```suggestion
      description: |-
```

File: .github/ISSUE_TEMPLATE/work.yaml [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+        What must be done for this work to be considered complete?
+        These can be
+      placeholder: "- [ ] e.g. x is done in y way so that z can be achieved"
+    validations:
+      required: false
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: |
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/work.yaml:36]:
```suggestion
      description: |-
```

File: .github/ISSUE_TEMPLATE/work.yaml [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+        What other work does this depend on? Enter the github issue
+        number in the form `- [ ] #356` or `- [ ] repo-name#243` to make
+        a checklist
```
Comment (paddyroddy) [inline .github/ISSUE_TEMPLATE/work.yaml:39]: Won't work without the org name
```suggestion
        number in the form `- [ ] #356` or `- [ ] UCL-MIRSG/repo-name#243` 
		to make a checklist
```

**Review by paddyroddy — CHANGES_REQUESTED** [review]
Looks good, stylistic changes mainly

File: .github/ISSUE_TEMPLATE/work.yaml (hunk 12 above)
Comment (HChughtai) [reply .github/ISSUE_TEMPLATE/work.yaml:39]: Thanks, good catch!

**Pull request merged**